lib/mix/test/mix/tasks/format_test.exs

```diff
@@ -174,6 +174,57 @@ defmodule Mix.Tasks.FormatTest do
     end
   end
 
+  defmodule FormatWithDepsApp do
+    def project do
+      [
+        app: :format_with_deps,
+        version: "0.1.0",
+        deps: [{:my_dep, "0.1.0", path: "deps/my_dep"}]
+      ]
+    end
+  end
+
+  test "can read exported configuration from dependencies" do
+    Mix.Project.push(__MODULE__.FormatWithDepsApp)
+
+    in_fixture "format_with_deps", fn ->
```

Comment:
If the fixture is empty you can use `in_tmp` instead.

Reply:
You're right, I didn't realize we were recreating the dependency directory structure in the test as well. Updated to use `in_tmp` :)